npm-debug.log

```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```

Comment:
Can you add this to `.gitignore` then remove it from the PR

Reply:
Just because everyone's npm-debug.log is going to be different and cause merge-conflicts